package.json

```diff
@@ -2,7 +2,7 @@
   "name": "@api3/operations",
   "license": "MIT",
   "private": false,
-  "version": "0.0.1-d",
+  "version": "0.0.1-93ec6619b89af6fa7e5feaf3f7347e3eb5a36b62",
```

Comment:
do you mind briefly explaining this change? 🙏🏻

Reply:
This was accidentally included.
I've been publishing ops for the Explorer team regularly and when I do I just add the git commit hash on the end to make the version unique.
We'll probably have to do something like this at a later stage via a Lambda handler (to protect credentials).

Reply:
npm package versions are immutable; once you've used a version you can't change the contents of that version.

Reply:
I've removed the hash :muscle: